src/scss/layout/_footer.scss

```diff
@@ -50,18 +50,14 @@ footer {
 				padding-bottom: rem(2);
 			}
 
-			.hewlett-href {
-				font-weight: $font-weight-normal;
-			}
-
 			.funder-logo {
 				display: inline-block;
 			}
 
 			svg {
-				height: rem(60);
-				padding: rem(13) rem(13) rem(13) 0;
-				width: rem(144);
+				height: rem(80);
```

Comment:
This will change the size of both hewlett and canada logos. I believe the decision from yesterday's meeting was to keep the canada logo at its original size and only increase the size of hewlett logo.

Reply:
Please add a class for the canada logo too and have line 57-61 apply to that class. It no longer makes sense to have that section applies to both hewlett and canada logos.